helm/frost-server/templates/_tplvalues.tpl

```diff
@@ -0,0 +1,38 @@
+{{/*
+Copyright Broadcom, Inc. All Rights Reserved.
```

Comment:
We can't just copy code with a difference license into the FROST repo.

Reply:
can we fix it when we use 

dependencies:
  - name: common
    version: 2.22.0
    repository: oci://registry-1.docker.io/bitnamicharts

Reply:
I think this would reference the helpers, developed [here](https://github.com/bitnami/charts/tree/main/bitnami/common). This might be a solution, but we need to discuss if we want to add the dependency only for two generic functions.

Reply:
Using a depencency is probably the cleanest and quickest solution. Could you make that change?